editor/components/PopupWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import { Popup } from "react-map-gl";
+import { TableMapPopupContentProps } from "./TableMapPopupContent";
+import { GeoJsonProperties } from "geojson";
+
+interface PopupWrapperProps {
+  latitude: number;
+  longitude: number;
+  featureProperties: GeoJsonProperties;
+  PopupContent: React.ComponentType<TableMapPopupContentProps>;
+  onClose: () => void;
+}
+
+export default function PopupWrapper({
+  latitude,
+  longitude,
+  featureProperties,
+  PopupContent,
+  onClose,
+}: PopupWrapperProps) {
+  return (
+    <Popup
+      latitude={latitude}
+      longitude={longitude}
+      // The popup won't render after multiple map feature click without this prop being false :/
```

Comment:
I copied this prop (and the comment) from the map configuration in the data and perfomance hub because I found the same issue

Reply:
yep so weird! i was able replicate without this prop so thanks for covering, Chia! 🙏